### PR TITLE
Feature styling update

### PR DESCRIPTION
**Pull request by MorisCamporesi**
### I make this draft PR to discuss my changes for issue: https://github.com/sopra-fs22-group-36/screw-your-neighbor-react/issues/13

All design choices were made for the final resolution of 1366x 768 as per the Projects requirements
I am aware that there will be merge conflicts with due to the changes done with the hamburger menu as well as my own tooltips.

_I feel like the code quality might not be up to par with previous work so feel free to give input!_

### CODE REVIEW

File: src/components/views/game/gamesummary/GameSummary.tsx [hunk 1]
```diff
@@ -25,27 +26,43 @@ export const GameSummary = observer(() => {
...
-          disabled={loading}
-          variant="contained"
-          endIcon={<SendIcon />}
-          onClick={clickStartNewGame}
-        >
-          Start new Game
-        </Button>
-      </BaseContainer>
+        <Box className="gameSummary-buttonBox">
+          <Button
```
Comment (CarmenUZH) [inline src/components/views/game/gamesummary/GameSummary.tsx:34]: >:(

**Comment by MorisCamporesi** [comment]
@CarmenUZH I think it would be good if we could pair for some last changes so we can find the spots we want to refine! :)

@all if you could just try to play one round / from registration to invite-link to playing to end just so you can quickly give me feedback :)

File: src/components/views/game/gamesummary/GameSummary.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
+@import 'src/styles/theme';
+
+.gameSummary-div{
+
+
+}
```
Comment (BacLuc) [inline src/components/views/game/gamesummary/GameSummary.scss:6]:
```suggestion
```

File: src/components/views/game/gamesummary/GameSummary.tsx [hunk 3]
```diff
@@ -2,14 +2,15 @@ import React from "react"
...
 import { extractId } from "../../../../util/extractId"
-import BaseContainer from "../../../ui/BaseContainer"
 import SendIcon from "@mui/icons-material/Send"
 import LogoutIcon from "@mui/icons-material/Logout"
 import Button from "@mui/material/Button"
 import { useNavigate } from "react-router-dom"
 import { Paths } from "../../../routing/routers/Paths"
 import ReactConfetti from "react-confetti"
 import { WinnersPodium } from "./WinnersPodium"
+import "../gamesummary/GameSummary.scss"
```
Comment (BacLuc) [inline src/components/views/game/gamesummary/GameSummary.tsx:12]:
```suggestion
import "GameSummary.scss"
```

File: src/components/views/game/gamesummary/WinnersPodium.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { useCurrentGame } from "../../../../hooks/api/useCurrentGame"
 import trophy from "../../../../img/winnerwinnerchickendinner.png"
 import lost from "../../../../img/betterluck.png"
 
-import "./WinnersPodium.scss"
+import "../gamesummary/GameSummary.scss"
```
Comment (BacLuc) [inline src/components/views/game/gamesummary/WinnersPodium.tsx:7]: ?
WinnersPodium should use WinndersPodium.scss, and not share the css with another component

File: src/components/views/game/gamesummary/WinnersPodium.scss [hunk 5]
```diff
@@ -1,57 +1 @@
-@import 'src/styles/theme';
```
Comment (BacLuc) [inline src/components/views/game/gamesummary/WinnersPodium.scss:1]: i would keep the css concerning WinnersPodium here

File: src/components/views/game/room/Room.scss [hunk 6]
```diff
@@ -1,15 +1,38 @@
 @import 'src/styles/theme';
 
+.room-div {
+  padding: 0 20px;
+}
 
-.room {
-  padding: 0 10px;
+.Button {
```
Comment (BacLuc) [inline src/components/views/game/room/Room.scss:7]: please use kebab-case

File: src/components/views/game/room/Room.scss [hunk 7]
```diff
@@ -1,15 +1,38 @@
...
+  padding: 20px;
 }
 
-.room-info-background{
+.room-player-div{
```
Comment (BacLuc) [inline src/components/views/game/room/Room.scss:11]:
```suggestion
.room-player-div {
```

File: src/components/views/game/room/Room.scss [hunk 8]
```diff
@@ -1,15 +1,38 @@
...
+
+  width: 100%;
+  display: block;
+  flex-direction: column;
+  background-color: #620606;
+  color: white;
+  border-radius: 2em;
```
Comment (BacLuc) [inline src/components/views/game/room/Room.scss:18]: i would reuse the $borderRadius from the theme

File: src/components/views/game/room/Room.scss [hunk 9]
```diff
@@ -1,15 +1,38 @@
...
-  border-radius: 10px;
+  overflow-y: scroll;
+  justify-content: space-around;
+  text-align: center;
+  margin: 20px auto;
+  min-height: 150px;
+  max-height: 400px;
 }
+
+.room-individual-player{
```
Comment (BacLuc) [inline src/components/views/game/room/Room.scss:28]:
```suggestion
.room-individual-player {
```

File: src/components/views/lobby/CreateGame.tsx [hunk 10]
```diff
@@ -10,6 +10,7 @@ import Box from "@mui/material/Box"
 import { Paths } from "../../routing/routers/Paths"
 import "../../../styles/ui/Box.scss"
 
+import "../lobby/Lobby.scss"
```
Comment (BacLuc) [inline src/components/views/lobby/CreateGame.tsx:13]: noo

File: src/components/views/lobby/Lobby.scss [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+  margin-right: auto;
+  display:block;
+  overflow-y: scroll;
+  max-height: 200px;
+  padding: 10px;
+  border-Radius: 1em;
+}
+
+
+.lobbyRooms{
```
Comment (BacLuc) [inline src/components/views/lobby/Lobby.scss:60]: kebab-case please

File: src/components/views/lobby/Lobby.scss [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+  font-size: 20px;
+  height: 50px;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  width: auto;
+  overflow: hidden;
+  background: red;
+  border-radius: 10px;
+  opacity: 100%;
```
Comment (BacLuc) [inline src/components/views/lobby/Lobby.scss:71]:
```suggestion
```

File: src/components/views/lobby/RoomRow.tsx [hunk 13]
```diff
@@ -8,7 +8,8 @@ import { extractId } from "../../../util/extractId"
 import { Grid, Tooltip } from "@mui/material"
 import Button from "@mui/material/Button"
 import { Login } from "@mui/icons-material"
-import "../../../styles/ui/Lists.scss"
+import "../lobby/Lobby.scss"
```
Comment (BacLuc) [inline src/components/views/lobby/RoomRow.tsx:11]: please don't share the Lobby.scss everywhere

File: src/components/views/game/room/RoomPlayerContainer.tsx [hunk 14]
```diff
@@ -3,7 +3,6 @@ import { observer } from "mobx-react-lite"
 import { useCurrentGame } from "../../../../hooks/api/useCurrentGame"
 import "../../../ui/BaseContainer.scss"
 import { Tooltip } from "@mui/material"
-import BaseContainer from "../../../ui/BaseContainer"
 
 export const RoomPlayerContainer = observer(() => {
```
Comment (BacLuc) [inline src/components/views/game/room/RoomPlayerContainer.tsx:7]: I don't know if it was also horrible before, but the disabled button is not readable

grafik

File: src/components/views/game/gamesummary/GameSummary.tsx [hunk 15]
```diff
@@ -2,14 +2,15 @@ import React from "react"
 import { observer } from "mobx-react-lite"
```
Comment (BacLuc) [inline src/components/views/game/gamesummary/GameSummary.tsx:2]: The view looks nice
grafik


I would put the number inside the li somehow, and align the player name left

**Review by BacLuc — CHANGES_REQUESTED** [review]

File: src/components/views/game/gamesummary/GameSummary.tsx (hunk 15 above)
Comment (MorisCamporesi) [reply src/components/views/game/gamesummary/GameSummary.tsx:2]: I am still trying to understand the exact workings as it looks differnt in different browsers (Chrome vs Firefox)

File: src/components/views/game/room/RoomPlayerContainer.tsx (hunk 14 above)
Comment (MorisCamporesi) [reply src/components/views/game/room/RoomPlayerContainer.tsx:7]: This is absolutely subject to change, I was not able to custom-color the @ mui Buttons how i wanted them but @CarmenUZH managed so I will sync with her to make it work

File: src/components/views/createplayer/CreatePlayer.scss [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+  font-family: 'Pacifico', cursive;
+  font-size: 40px;
+  text-align: center;
+}
+
+.registerBoxTitle {
+  text-align: center;
+  font-family: 'Pacifico', cursive;
+  font-size: 40px;
+  color: whitesmoke;
```
[the remark this answers is not in the document]
Comment (MorisCamporesi) [reply src/components/views/createplayer/CreatePlayer.scss]: mistake, is fixed

File: src/components/views/game/gamesummary/GameSummary.tsx [hunk 17]
```diff
@@ -1,51 +1,41 @@
...
-import { useGames } from "../../../../hooks/api/useGames"
-import { extractId } from "../../../../util/extractId"
-import BaseContainer from "../../../ui/BaseContainer"
-import SendIcon from "@mui/icons-material/Send"
-import LogoutIcon from "@mui/icons-material/Logout"
-import Button from "@mui/material/Button"
-import { useNavigate } from "react-router-dom"
-import { Paths } from "../../../routing/routers/Paths"
 import ReactConfetti from "react-confetti"
```
Comment (BacLuc) [inline src/components/views/game/gamesummary/GameSummary.tsx:3]: It currently is too large for the 1366x768 resolution:
grafik

Comment (BacLuc) [reply src/components/views/game/gamesummary/GameSummary.tsx:3]: Fixed with https://github.com/sopra-fs22-group-36/screw-your-neighbor-react/pull/129/commits/c82230369afddc98d00cf7fe04c3b76dc826e856

**Review by BacLuc — APPROVED** [review]

File: src/components/views/game/gamesummary/GameSummary.tsx (hunk 17 above)
Comment (MorisCamporesi) [reply src/components/views/game/gamesummary/GameSummary.tsx:3]: yes it was broken in the merge/rebase from the other branch. I am currently working on fixing it

Comment (MorisCamporesi) [reply src/components/views/game/gamesummary/GameSummary.tsx:3]: I did a small rework on the branch.
this is how it looks now
Bildschirmfoto am 2022-05-28 um 14 55 22

**Pull request merged**